app/(admin)/admin/upload-spreadsheet/UploadSpreadsheetClient.tsx

```
@@ -0,0 +1,1281 @@
+      case "year":
+        if (typeof val === "number" || typeof val === "string")
+          mapped.year = String(val);
+        break;
+      case "email":
+      case "Email":
+        mapped.email = asStr(val).toLowerCase().trim();
+        break;
+      case "name":
+        mapped.name = asStr(val);
+        const parts = mapped.name.split(" ");
+        if (!mapped.firstName) mapped.firstName = parts[0] || "";
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)